src/core/mod.rs

```diff
@@ -452,7 +460,7 @@ impl Tox {
     pub fn wait(&self) {
         unsafe {
             let delay = ll::tox_iteration_interval(self.raw);
-            sleep_ms(delay);
+            sleep(Duration::new(0, delay));
```

Comment:
Why change from `sleep_ms()` to `sleep()` ?

According to [Rust docs](https://doc.rust-lang.org/stable/std/time/struct.Duration.html), `Duration` takes as argument nanoseconds, whereas `tox_iteration_interval()` [returns milliseconds](https://github.com/irungentoo/toxcore/blob/0938ba08d9a839df4111168f3414099268d69737/toxcore/tox.h#L782,L785).